rfcs/20210510-conda-forge-distribution/README.md

```diff
@@ -0,0 +1,52 @@
+# Conda-forge distribution
+
+## Introduction
+
+Conda is a python package manager that widely used by ML FWKs users
+
+## Conda channels
+- default. This is a channel managed by Anaconda core team.
+  oneDNN is available as part of Intel oneAPI [distribution](https://anaconda.org/intel/repo).
+- forge. Conda forge is a community led Conda channel that consists of build
+  infrastructure and distributions for the Conda package manager. Recently oneDNN became
+  available in [Conda-forge](https://github.com/conda-forge/onednn-feedstock) channel.
+
+The main differences between Conda and Conda-forge:
+- Conda-forge has broader architecture support;
+- Conda-forge has more packages available;
+- To provide binary compatibility between packages in Conda-forge all package
+  dependencies should be packaged by Conda-forge, meaning there is no way to
+  distribute oneDNN built by Intel C++ Compiler without having this compiler
+  in Conda-forge;
+- And [more](https://conda-forge.org/docs/user/introduction.html#why-conda-forge).
+
+## oneDNN usage cases. Summary
+
+|          | Runtime package        | Build time package     | Full bundle  |
+| :------- | :--------------------- | :--------------------- | :----------- |
+| Scenario | Used as a dependency for binary distributions | Used as a dependency for source code distributions that are dependent on oneDNN. | Covers remaining cases |
+| Channel  | **conda**, pip, OS repos | **conda**, pip, OS repos | Intel oneAPI |
+| Library  | x                      | x                      | x            |
```

Comment:
Do we have to make any changes in the build system to teach it how to install only lib or only lib and headers, or this is completely handled by conda-forge?

Reply:
Since we can modify scripts which build and package oneDNN on feedstock side, this can be handled by in conda-forge. Actually I tried to install oneDNN in conda and it contains headers, so current conda-forge builds are already devel. I will update the RFC accordingly.